src/Symfony/Component/Config/Tests/FileLocatorTest.php

```diff
@@ -115,6 +115,6 @@ public function testLocateEmpty()
     {
         $loader = new FileLocator([__DIR__.'/Fixtures']);
 
-        $loader->locate(null, __DIR__);
+        $loader->locate('', __DIR__);
```

Comment:
This test was expecting an `InvalidArgumentException` but gets a `TypeError` now.